### PR TITLE
refactor: change message

### PR DESCRIPTION
**Pull request by ace-e4s**

### CODE REVIEW

File: RepoDemo/Program.cs [hunk 1]
```diff
@@ -6,7 +6,7 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Commit the World!");
+            Console.WriteLine("Goodbye Cruel World!");
```
Comment (peter-lillevold-4ss) [inline RepoDemo/Program.cs]: Aaaw

Comment (ace-e4s) [reply RepoDemo/Program.cs]: No way!

Comment (ingar-hillestad-4ss) [reply RepoDemo/Program.cs]: Dont think its hard enough

File: RepoDemo/Program.cs [hunk 2]
```diff
@@ -6,7 +6,7 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Commit the World!");
```
Comment (peter-lillevold-4ss) [inline RepoDemo/Program.cs]: Multiline iz nize

**Pull request closed without merging**